exporters/zipkin/zipkin_test.go

```diff
@@ -324,6 +325,29 @@ func TestExportSpans(t *testing.T) {
 	}
 	require.Eventually(t, checkFunc, time.Second, 10*time.Millisecond)
 	require.Equal(t, models, collector.StealModels())
+
+	got := make(chan *http.Request, 1)
```

Comment:
This should be a new test, not part of an existing one.

Reply:
Updated the test, please check.